cmd/containerd-shim-runhcs-v1/service_internal.go

```diff
@@ -68,6 +69,14 @@ func (s *service) stateInternal(ctx context.Context, req *task.StateRequest) (*t
 	return e.Status(), nil
 }
 
+func updateSpecFromOptions(s specs.Spec, opts *runhcsopts.Options) specs.Spec {
```

Comment:
I think I should move this into uvm.go so `AnnotationBootFilesRootPath` doesn't need to be exported.